src/Analyzers/Core/Analyzers/Formatting/FormattingAnalyzerHelper.cs

```diff
@@ -8,12 +8,8 @@
 using Microsoft.CodeAnalysis.Formatting;
 using Microsoft.CodeAnalysis.Text;
 
-#if CODE_STYLE
 using Formatter = Microsoft.CodeAnalysis.Formatting.FormatterHelper;
 using FormattingProvider = Microsoft.CodeAnalysis.Formatting.ISyntaxFormatting;
-#else
-using FormattingProvider = Microsoft.CodeAnalysis.Host.HostWorkspaceServices;
-#endif
```

Comment:
Previously, `Features` was using Mef and `CodeStyle` was using `ISyntaxFormatting`. Now both are using `ISyntaxFormatting`. @mavasani Can you confirm this is the right action?

Reply:
I think it should be fine, @sharwell to confirm